### PR TITLE
Enable preview of model based eq #67

### PR DESCRIPTION
**Pull request by longshuicy**
To test:
- run `npm run start:dev`
- Hazard Viewer --> Earthquake --> any model based eq
- Find button in "rasterDataset" cell:
image

Note not all preview works, some of the layer doesn't exist on dev geoserver for some reason. But that's beyond the scope of this PR.

### CODE REVIEW

**Comment by ywkim312** [comment]
please update changelog

File: src/components/children/NestedInfoTable.jsx [hunk 1]
```diff
@@ -52,8 +50,8 @@ class NestedInfoTable extends React.Component {
 														return (<div key={hazardDataset.datasetId}>
 															<ListItem key={hazardDataset.datasetId}>
 																<Button color="primary" variant="contained"
-																	size="small" className={classes.inlineButtons}
-																	onClick={() => this.props.onClick(hazardDataset.datasetId)}>Preview</Button>
+																		size="small" className={classes.inlineButtons}
```
Comment (navarroc) [inline src/components/children/NestedInfoTable.jsx:53]: Can you fix the eslint errors? I see a few for this file "error Expected indentation of 17 tabs but found 18  indent"

**Review by ywkim312 — APPROVED** [review]
I don't know too much about react. I ran, it worked. I approve

File: src/components/children/NestedInfoTable.jsx (hunk 1 above)
Comment (longshuicy) [reply src/components/children/NestedInfoTable.jsx]: Reformated

**Review by navarroc — APPROVED** [review]

**Pull request merged**